### ISSUE TITLE
Parse timeout response

### ISSUE THREAD

**Issue by alexkwolfe** [activeprospect/leadconduit-integration-velocify#46]
Sometimes Velocify gives a timeout response. This should be interpreted as an error outcome with reason "Velocify timeout". The timeout response looks like this:

```
HTTP/1.1 200 OK
Cache-Control: private
Content-Type: text/xml; charset=utf-8
Server: Microsoft-IIS/7.5
Set-Cookie: ["ASP.NET_SessionId=oeul5g4qoftkvm1w2sj2im3t; path=/; HttpOnly"]
X-Aspnet-Version: 4.0.30319
X-Powered-By: ASP.NET
Date: Wed, 22 Jul 2015 18:23:16 GMT
Content-Length: 247

 <?xml version="1.0" encoding="UTF-8"?><ImportResults><ImportResult refId="" leadId="0" result="Failure" message="Timeout expired.  The timeout period elapsed prior to completion of the operation or the server is not responding." /></ImportResults>
```

**Comment by alexkwolfe** [comment]
This response is currently interpreted as an error with the "Unsupported Velocify response" reason.

**Comment by zalary** [comment]
resolves with Version 0.0.18
